pkg/pillar/scripts/device-steps.sh

```diff
@@ -425,14 +425,21 @@ if [ ! -f $CONFIGDIR/device.cert.pem ]; then
     sync
     blockdev --flushbufs "$CONFIGDEV"
     if [ -c $TPM_DEVICE_PATH ] && ! [ -f $CONFIGDIR/disable-tpm ]; then
-        echo "TPM device is present and allowed, marking mode as tpm-enabled"
-        touch $PERSISTCONFIGDIR/tpm_in_use
-        sync
-        blockdev --flushbufs "$CONFIGDEV"
-        $BINDIR/generate-device.sh -b $CONFIGDIR/device -t
+        echo "TPM device is present and allowed, creating TPM based device key"
```

Comment:
Since you're touching this can you change it to log with the data i.e.,
echo "$(date -Ins -u) ..."

Reply:
Done